ayekan/fluentbit/namespace.yaml

```diff
@@ -0,0 +1,9 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: fluent
+  labels:
+    lsst.org/author: "fbegyn"
+    lsst.org/discover: "true"
+    lsst.org/monitor: "true"
```

Comment:
Is this replacing `o11y.eu/monitor: enabled` ?

Reply:
I was playing around with some label formats on an experimental branch, apparently streams go crossed somewhere.

Reply:
we've decided on an `lsst.io/<>` scheme. Moving forward to it.